src/main.cpp

```diff
@@ -3851,10 +3854,40 @@ void UnlinkPrunedFiles(std::set<int>& setFilesToPrune)
     }
 }
 
+/* Calculate the block/rev files to delete based on height specified by user with RPC command pruneblockchain */
+void FindFilesToPruneManual(std::set<int>& setFilesToPrune, int nManualPruneHeight)
+{
+    LOCK2(cs_main, cs_LastBlockFile);
+    if (chainActive.Tip() == NULL || pruneMode != PRUNE_MANUAL) {
```

Comment:
Calling this function with pruneMode != PRUNE_MANUAL should be an assertion error, as it must be a bug in the code.